spec/controllers/invite_requests_controller_spec.rb

```diff
@@ -179,16 +179,45 @@
     end
 
     context "when logged in as admin" do
+      let(:ip) { "127.0.0.1" }
       let!(:invite_request_1) { create(:invite_request, position: 9001) }
       let!(:invite_request_2) { create(:invite_request, position: 2) }
       let!(:invite_request_3) { create(:invite_request, position: 7) }
+      let!(:invite_request_4) do
+        create(
+          :invite_request,
+          position: 500,
+          email: "hello_world@gmail.com"
+        )
+      end
 
-      before { fake_login_admin(admin) }
+      before do
+        fake_login_admin(admin)
+      end
+
+      it "searches invitations by email" do
+        get :manage, params: { query: "hello_world" }
+        expect(response).to render_template("manage")
+        expect(assigns(:invite_requests)).to eq([invite_request_4])
+      end
+
+      it "searches invitations by ip" do
```

Comment:
Can you set a different IP on one of the other 3 requests? Then this spec would make sure that other IP won't show up here.